plugins/Files/js/components/renamedialog.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react'
+
+const RenameDialog = ({siapath, actions}) => {
+	const onYesClick = (e) => {
+		e.preventDefault()
+		actions.renameFile(siapath, e.target.newsiapath.value)
+		actions.hideRenameDialog()
```

Comment:
Note that this will cause the rename dialog to hide immediately after the user clicks yes to rename the file.  The alternative is to place `actions.hideRenameDialog` inside the `renameFile` saga, such that the dialog is only hidden when a file is successfully renamed.  I think I'd rather have it be in the saga, but the rest of the team's input on this would be helpful.